modules/pcd/src/lib/parse-pcd.ts

```diff
@@ -93,14 +141,21 @@ function getNormalizedAttributes(attributes) {
   return normalizedAttributes;
 }
 
+/**
+ * Incoming data parsing
+ * @param data
+ * @returns Header
+ */
 /* eslint-disable complexity, max-statements */
-function parsePCDHeader(data) {
+function parsePCDHeader(data: string): IncomingHeader {
```

Comment:
Can this return type be called `PCDHeader`

Reply:
> Can this return type be called `PCDHeader`

changed